src/code/gfxprint.c

```diff
@@ -222,21 +222,23 @@ void GfxPrint_PrintCharImpl(GfxPrint* this, u8 c) {
 
         if (this->flag & GFXPRINT_FLAG64) {
             gSPTextureRectangle(this->dlist++, (this->posX + 4) << 1, (this->posY + 4) << 1, (this->posX + 4 + 32) << 1,
-                                (this->posY + 4 + 32) << 1, tile, (u16)(c & 4) * 64, (u16)(c >> 3) * 256, 512, 512);
+                                (this->posY + 4 + 32) << 1, tile, (u16)(c & 4) * 64, (u16)(c >> 3) * 256, (1 << 10) / 2,
+                                (1 << 10) / 2);
```

Comment:
Not too sure about the few `(1 << 10) / 2` there. I'm not too knowledgeable about this stuff but I'm pretty sure that's not how it's supposed to be (referring specifically to the `/ 2`, not the `(1 << 10)`s)